### PR TITLE
qa/tasks: Fix AttributeError: can't set attribute

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: https://tracker.ceph.com/issues/42636

### CODE REVIEW

**Comment by joscollin** [comment]
http://pulpito.front.sepia.ceph.com/jcollin-2019-11-06_12:50:58-fs-wip-thrasher-testing2-distro-basic-smithi/

**Comment by joscollin** [comment]
> (batrick) use -s fs:thrash instead

@batrick Looks Good.

http://pulpito.front.sepia.ceph.com/jcollin-2019-11-07_04:45:11-fs:thrash-wip-thrasher-testing2-distro-basic-smithi/

File: qa/tasks/rbd_mirror_thrash.py [hunk 1]
```diff
@@ -87,7 +86,7 @@ def _run(self):
             self.do_thrash()
         except Exception as e:
             # See _run exception comment for MDSThrasher
-            self.exception = e
+            super(RBDMirrorThrasher, self.__class__).exception.fset(self, e)
```
Comment (batrick) [inline qa/tasks/rbd_mirror_thrash.py]: I feel like this is needlessly complicated: why not just define a method in `Thrasher`:

````
def set_thrasher_exception(self, e):
    self._exception = e
````

and then here you can do:

`self.set_thrasher_exception(e)`

Manipulating the members of the parent class directly like this is a code smell IMO.

File: qa/tasks/thrasher.py [hunk 2]
```diff
@@ -15,11 +14,3 @@ def exception(self):
     @exception.setter
     def exception(self, e):
         self._exception = e
-
-    @property
-    def name(self):
-        return self._name
```
Comment (batrick) [inline qa/tasks/thrasher.py]: You can keep this property method but raise a `NotImplementedError`.

**Comment by joscollin** [comment]
@batrick Fixed

**Comment by joscollin** [comment]
http://pulpito.front.sepia.ceph.com/jcollin-2019-11-19_10:54:49-fs-wip-42636-attribute-error-testing-distro-basic-smithi/

**Comment by joscollin** [comment]
jenkins test make check

**Comment by joscollin** [comment]
http://pulpito.front.sepia.ceph.com/jcollin-2019-11-27_06:35:12-fs:thrash-wip-42636-attribute-error_testing-distro-basic-smithi/

**Pull request merged**